test/test_aa.py

```diff
@@ -519,6 +519,35 @@ def testPermittedClustersOnly(self):
                 self.assertIn(clusterNr, {1, 2, 3, 4, 5})
 
 
+class TestPropertyDetailsRaw(TestCase):
+    """
+    Tests for the PROPERTY_DETAILS_RAW dict.
+    """
+    def testPropertyDetailsRawKeys(self):
+        """
+        The PROPERTY_DETAILS_RAW dict must contain the right keys.
+        """
+        self.assertEqual(AA_LETTERS, sorted(PROPERTY_DETAILS_RAW.keys()))
+
+    def testNumberOfValues(self):
+        """
+        Each key in PROPERTY_DETAILS_RAW must have a dict with 10 elements in
+        it as the value.
+        """
+        for propertyNames in PROPERTY_DETAILS_RAW.values():
+            self.assertEqual([
+                'aliphaticity', 'aromaticity', 'composition',
+                'hydrogenation', 'hydropathy', 'hydroxythiolation',
+                'iep', 'polar requirement', 'polarity', 'volume'],
+                sorted(propertyNames.keys()))
```

Comment:
You don't need the `.keys()` here. When you iterate over a `dict`, you get its keys. That's probably my Python :-)